pkg/agent/testdata/AKSUbuntu1604+CustomKubeletConfig+CustomLinuxOSConfig/line16.sh

```diff
@@ -2,10 +2,16 @@
 EXIT_CODE=$?
 systemctl --no-pager -l status kubelet >> /var/log/azure/cluster-provision-cse-output.log 2>&1
 OUTPUT=$(cat /var/log/azure/cluster-provision-cse-output.log | head -n 30)
+START_TIME=$(echo "$OUTPUT" | cut -d ',' -f -1 | head -1)
+CSE_EXECUTION_DURATION=$(echo $(($(date +%s) - $(date -d "$START_TIME" +%s))))
+echo "CSE Total Execution Duration:$CSE_EXECUTION_DURATION  seconds" >> /var/log/azure/cluster-provision-execution-durations.log 2>&1
```

Comment:
the total duration should be a separate property in the JSON_STRING. The content of /var/log/azure/cluster-provision-execution-durations.log cannot be directly put into json_string as the json_string has limited length. It's also not parsable by RP. Let's only expose the total time for now.

Reply:
"CSEExecutionDurations": "cleanUpContainerd:0 seconds\ncleanUpGPUDrivers:0 seconds\ncleanUpContainerImages:0 seconds\ninstallContainerRuntime:2 seconds\ninstallNetworkPlugin:7 seconds\ninstallKubeletKubectlAndKubeProxy:0 seconds\ncreateKubeManifestDir:0 seconds\nconfigureK8s:0 seconds\nconfigureCNI:1 seconds\nensureDocker:23 seconds\nensureMonitorService:0 seconds\nensureSysctl:0 seconds\nensureKubelet:1 seconds\nensureJournal:1 seconds\nensureUpdateNodeLabels:0 seconds\nCSE Total Execution Duration:39 seconds"

Reply:
also cluster-provision-cse-output.log and the other new log file should be downloadable from asc
https://github.com/Azure/azure-diskinspect-service